infrastructure/resource_group_level/main.bicep

```diff
@@ -290,6 +290,13 @@ var parametersMap = {
       storageNetworkAccessDefaultAction: 'Allow'
     }
     apiPortalAccessPort: 44232
+    redis: {
```

Comment:
I've just moved this into the parametersMap to make it clear we don't have to have the same values.